lib/MusicBrainz/Server/Data/Entity.pm

```diff
@@ -60,11 +48,18 @@ sub get_by_id
     return $result[0];
 }
 
-sub get_by_id_locked
-{
+sub get_by_id_locked {
```

Comment:
The function not only gets the IDs, but also puts a row lock on them via the `FOR UPDATE` clause. See http://www.postgresql.org/docs/9.1/static/explicit-locking.html#LOCKING-ROWS